Components/Toast/toast.html

```diff
@@ -0,0 +1,128 @@
+</head>
+
+<body>
+    <header class="header-container">
+        <h1 class="header-heading"><a class="logo" href="/index.html">Vision UI</a></h1>
+        <div class="header-link">
+            <p class="home-link"><a href="/index.html">Home</a></p>
+        </div>
+        </div>
+        <div class="headerlink-nav">
+            <a href="https://github.com/sumitch29"><i class="fab fa-github font-xl headerlink-nav-icon"></i></a>
+            <a href="https://twitter.com/sumit__chavhan"><i class="fab fa-twitter font-xl headerlink-nav-icon"></i></a>
+            <a href="https://www.linkedin.com/in/sumit-chavhan/"><i
+                    class="fab fa-linkedin font-xl headerlink-nav-icon"></i></a>
+        </div>
+    </header>
+    <main>
+        <nav>
+            <ul class="sidebar">
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/index.html">Get Started</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Avatar/avatar.html">Avatar</a>
+                </li>
+                <li class="sidebar-link active">
+                    <a class="sidebarlink" href="/Components/Alert/alert.html">Alert</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Badge/badge.html">Badge</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Button/button.html">Button</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Card/card.html">Card</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Image/image.html">Image</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Input/input.html">Input</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Text-Utilities/text.html">Text Utilities</a>
+                </li>
+                <li class="sidebar-link"><a class="sidebarlink" href="/Components/List/list.html">Lists</a></li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Rating/rating.html">Rating</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/Grid/grid.html">Grid</a>
+                </li>
+                <li class="sidebar-link">
+                    <a class="sidebarlink" href="/Components/modal/modal.html">Modal</a>
+                </li>
+                <li class="sidebar-link active">
+                    <a class="sidebarlink" href="/Components/Toast/toast.html">Snackbar</a>
```

Comment:
name files as per component. (easy to remember)

Reply:
Thanks for pointing out. I will change it.